aten/src/ATen/native/metal/MetalPrepackOpRegister.cpp

```diff
@@ -102,6 +102,99 @@ Tensor conv2d_prepack_run(
 #endif
 }
 
+
+c10::intrusive_ptr<Conv3dOpContext> unpack(
```

Comment:
had a redefinition error here: "redefinition of 'unpack'"

Reply:
Thanks. Adapting the names...